src/v/kafka/protocol/schemata/generator.py

```diff
@@ -883,21 +951,19 @@ def is_default_comparable(self):
 
 HEADER_TEMPLATE = """
 #pragma once
-#include "kafka/types.h"
 #include "kafka/protocol/types.h"
 #include "model/fundamental.h"
 #include "model/metadata.h"
-#include "kafka/protocol/batch_reader.h"
 #include "kafka/protocol/errors.h"
-#include "model/timestamp.h"
 #include "seastarx.h"
 
-#include <seastar/core/sstring.hh>
-
-#include <chrono>
-#include <cstdint>
-#include <optional>
-#include <vector>
+{%- for header in struct.headers("header") %}
+{%- if header.startswith("<") %}
```

Comment:
nit: This conditional can be removed if the values in the dictionary that are project included files just come with quotes included.

Reply:
considered that originally, but that just adds more work for some editing the `extra_headers` since they have to make sure they escape their quotes correctly, etc... i would definitely not care if you changed this later to eliminate the loop.